ArcGISRuntimeSDKQt_CppSamples/Geometry/FormatCoordinates/FormatCoordinates.cpp

```diff
@@ -180,7 +180,7 @@ void FormatCoordinates::setMapView(MapQuickView* mapView)
     // using the point, refresh the graphic and the text
     handleLocationUpdate(std::move(mapPoint));
   },
-  Qt::UniqueConnection);
+  Qt::UniqueConnection); // Finds usages of Qt::UniqueConnection when the slot is a functor, lambda or non-member function. That connect() overload does not support Qt::UniqueConnection.
```

Comment:
I didn't know about that. I guess we should create an issue to work on that.

Reply:
I guess the best is to remove that, as this is ignored for the moment.

Reply:
The samples team needs to follow up on this. If the logic of the sample depends on this only ever being connected once, then it will not work correctly. @tanneryould please follow up with @GuillaumeBelz and log an issue if you haven't already.